app/src/main/java/com/example/unlibrary/library/LibraryRepository.java

```diff
@@ -268,7 +275,11 @@ public void fetchRequestersForCurrentBook(String currentBookID) {
      * @param bookID requests on this book will be listened to
      */
     public void attachRequestsListener(String bookID) {
-        Query query = mDb.collection(REQUESTS_COLLECTION).whereEqualTo(BOOK, bookID).whereNotEqualTo(STATE, Request.State.ARCHIVED);
+        // Get all requests associated with current book that are in REQUESTED state
+        Query query = mDb.collection(REQUESTS_COLLECTION)
+                .whereEqualTo(BOOK_FIELD, bookID)
+                .whereEqualTo(STATE_FIELD, Request.State.REQUESTED.name());
```

Comment:
```suggestion
                .whereEqualTo(STATE_FIELD, Request.State.REQUESTED.toString());
```
According to [ref](https://dzone.com/articles/enum-using-the-name-and-tostring-methods-correctly) although .name() and toString() return essentially the same thing, use of `toString()` is encouraged and we're already using this everywhere else

Reply:
name() would convert the enum to string exactly like the enum
toString() can be overridden and its behavior can be changed.
If we use toString() everywhere, we should be consistent. I'll update this

Reply:
Thanks!

Reply:
Done